templates/getting_started_python/src/main.py

```diff
@@ -0,0 +1,33 @@
+import asyncio
+
+from apify import Actor
+
+
+async def main():
+    async with Actor() as actor:
+        # Get the value of the actor input
+        actor_input = await actor.get_input()
+
+        # Structure of input is defined in INPUT_SCHEMA.json
+        first_number = getattr(actor_input, "first_number", None)
```

Comment:
```suggestion
        first_number = actor_input.get("first_number")
```

^ this is equal to `actor_input["number"]` with None as default. `getattr` is for `actor_input.property`, which isn't the same in Python.

the disadvantage is that it will throw an error if input is None - so maybe let's also add
```py
if type(actor_input) != dict:
    raise ValueError("Input must be a dictionary")
```

before this line? (or maybe even add a hint that the input should be in the storage/kv/default/input.json, unless that is handled by the SDK)

Reply:
This was done precisely so that it doesn't throw an error when input is None and does not fail on first run, before you create the storages and so on. I'd like to have some default input in the templates, at least for these "getting started" ones, but I think it was removed a few months back, I need to ask around why.

Reply:
Sorry, I didn't write it well, but what I wanted to say that this won't work when the input is provided - the loaded json is a dictionary, but getattr() (= `x.prop`) doesn't access dictionary fields (`x["field"]`)

Reply:
Riiiight, I need to read things better. 🤦

Now I wonder - should `Actor.get_input()` always return something? If there is no `INPUT` key in the default key-value store, return an empty dict? It would make it a lot easier to work with.

Reply:
I don't think we should fallback to empty dict.

In JS SDK, we return undefined when there's no input, so by following the same logic, in Python it can return None. OTOH, Python and JS differ in what is considered "idiomatic" - in Python, many things (such as accessing fields of dictionary with `x['field']`) raise an error, that you can catch. While in JS, exceptions often only used for really critical issues, and when something just doesn't exist, you get `undefined`. So by following this, I would even support throwing an exception directly in `get_input` when there isn't any.

Reply:
I think it's actually valid that a run is started with no input, and so it should not throw an exception.

But I agree, let's keep returning `None` in that case, and then just do `input = Actor.get_input() or {}` in the template, so that it's explicit, but easier to work with down the road.

Reply:
> I'd like to have some default input in the templates, at least for these "getting started" ones, but I think it was removed a few months back, I need to ask around why.

@fnesveda Because the platform does not respect input in local KVS and locally the actor does not respect input schema prefills and defaults. So it was just confusing people and we decided to get rid of the local inputs. But we need to find a better way to do this. Perhaps update CLI. Or have SDK itself check and validate input schema when running locally.

Reply:
Maybe the CLI could check if there is any input schema, and if there are any defaults in it, and if there are, generate a `INPUT` key in the emulated default key-value store with those defaults. I think it would make the templates much easier to work with for beginners.

We have to have a function for generating the default input from input schema ready somewhere, the platform does it too.

Reply:
> get rid of local inputs

@mnmkng sorry, what does that mean?

@fnesveda automatically providing input schema default if there's not input seems like too a bit too much magic to me. But it might be actually good if it wrote this default into the storage, so the next time they are there and user can edit it.

Reply:
@mvolfik most templates used to have `/storage/key-value-stores/default/INPUT.json` file and some default input in it. But it was confusing users and we had a few related bugs in CLI, so we removed the default inputs from most templates. But maybe there still are some, I don't know and don't wanna check all of them 😅